library/src/skins/default/src/sass/modules/tool/gradebook/_gradebook.scss

```diff
@@ -532,6 +532,9 @@
   .handsontable th {
     overflow: visible; /* to allow visual cue to be rendered nicely */
   }
+  .handsontable th .hide {
```

Comment:
We shouldn't need to be doing this with bs-5. We can just use d-none on the elements rather than have our own class. I think what probably happened here is that a previous class .hide was removed from the gb styles, but d-none was not fully implemented.

Reply:
Fwiw, the .hide is a class from handsontable, that for some reason within the Trinity context, is being superseded by another selector that affects the same span. I think I can append a d-none to the span, but whether or not to then get rid of the existing 'hide' remains to be seen (though likely a matter of code clean-up aesthetics).

I'll begin with appending a d-none class and test if that works without this additional CSS statement.